src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  const NUMBER_OF_ATTEMPS = 3;
+  let counter = 0;
+
+  return (firstNumber) => {
+    return (secondNumber) => {
+      counter++;
+
+      if (counter > NUMBER_OF_ATTEMPS) {
+        counter -= 2;
```

Comment:
```suggestion
        counter = 0;
```
Use number_of_attempts variable instead of the hardcoded number. Or, even better, just reset it